### ISSUE TITLE
[Bug]: Plugin dropdown_input does not make AT announce options

### ISSUE THREAD

**Issue by JoackimPennerup** [orchidjs/tom-select#672]
### Bug description

When Tom-select is used with _dropdown_input_, NVDA does not announce the selection, instead is says "blank". This means I'm unable to understand what is selected when using assistive technology (AT). 
I think the `aria-activedescendant` needs to be on the focused element in order for the AT to understand that it should annonce something different than the currently focused element.

### Expected behavior

When not using dropdown_input, options are announced as "Option text  1 of n". 

Example output from NVDA on an example without dropdown_input:
> Basic Examples - Tom Select  document
> list
> Thomas Edison  4 of 4
> Nikola Tesla  3 of 4

### Steps to reproduce

1. Go to https://tom-select.js.org/plugins/dropdown-input/
2. Focus the tom-select with keyboard while NVDA is running.
3. NVDA announces the combobox
4. Use arrow keys up and down. 
5. NVDA annonces "blank" multiple times.

### Additional context

- Windows 11
- Chrome 120
- NVDA 2023.3
